<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ui_CPicFlowWidget</class>
 <widget class="QWidget" name="Ui_CPicFlowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(144, 178, 196)</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="LeftArrowFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="LeftArrow">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>128</horstretch>
          <verstretch>128</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image:url(:/Images/left4.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="CenterFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="CPictureFlowWidget" name="PictureFlow">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string>This is Picture Flow, an image display plugin.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="ScrollBar">
        <property name="styleSheet">
         <string notr="true">QScrollBar:horizontal { 
	border-image: url(:/Images/scrollbar_h_bg.png) 0 4 0 4 stretch; 
	height: 15px; 
	margin: 0 15px 0 15px; 
	padding-left:-5px; 
	padding-right:-5px; 
	border-width: 0px 4px 0px 4px;
} 
QScrollBar::handle:horizontal { 
	border-image: url(:/Images/scrollbar_h_body.png) 0 4 0 4 stretch; 
	min-width: 20px; 
	border-width: 0px 4px 0px 4px;
} 
QScrollBar::handle:horizontal:hover { 
	border-image: url(:/Images/scrollbar_h_body_hover.png)0 4 0 4 stretch; 
	border-width: 0px 4px 0px 4px;
} 
QScrollBar::handle:horizontal:pressed { 
	border-image: url(:/Images/scrollbar_h_body_pressed.png) 0 4 0 4 stretch; 
	border-width: 0px 4px 0px 4px;
} 
QScrollBar::sub-line:horizontal { 
	border: 0px; 
	background: none; 
	width: 15px; 
	subcontrol-position: left; 
	subcontrol-origin: margin;
} 
QScrollBar::add-line:horizontal { 
	border: 0px; 
	background: none; 
	width: 15px; 
	subcontrol-position: right; 
	subcontrol-origin: margin;
} 
QScrollBar::left-arrow:horizontal { 
	background-image: url(:/Images/scrollbar_h_leftArrow.png); 
	background-repeat: no repeat; 
	width: 15px; 
	height: 15px;
} 
QScrollBar::left-arrow:horizontal:hover { 
	background-image: url(:/Images/scrollbar_h_leftArrow_hover.png); 
	background-repeat: no repeat; 
	width: 15px; 
	height: 15px;
} 
QScrollBar::left-arrow:horizontal:pressed { 
	background-image: url(:/Images/scrollbar_h_leftArrow_pressed.png); 
	background-repeat: no repeat; 
	width: 15px; 
	height: 15px;
} 
QScrollBar::right-arrow:horizontal { 
	background-image: url(:/Images/scrollbar_h_rightArrow.png); 
	background-repeat: no repeat; 
	width: 15px; 
	height: 15px;
} 
QScrollBar::right-arrow:horizontal:hover { 
	background-image: url(:/Images/scrollbar_h_rightArrow_hover.png); 
	background-repeat: no repeat; 
	width: 15px; 
	height: 15px;
} 
QScrollBar::right-arrow:horizontal:pressed { 
	background-image: url(:/Images/scrollbar_h_rightArrow_pressed.png);
	background-repeat: no repeat; 
	width: 15px; 
	height: 15px;
} 
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal { 
	background: none;
} 
</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="RightArrowFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="RightArrow">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>50</horstretch>
          <verstretch>50</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/Images/right4.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPictureFlowWidget</class>
   <extends>QWidget</extends>
   <header>CPictureFlowWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="UiImgResources.qrc"/>
 </resources>
 <connections/>
</ui>
